src/UglyToad.PdfPig/Content/PageContent.cs

```diff
@@ -23,15 +24,17 @@ internal class PageContent
         internal IReadOnlyList<IGraphicsStateOperation> GraphicsStateOperations { get; }
 
         public IReadOnlyList<Letter> Letters { get; }
+        public List<PdfPath> Paths { get; }
```

Comment:
Same here, I'd change this to:
```suggestion
        public IReadOnlyList<PdfPath> Paths { get; }
```